services/OrderService.py

```
@@ -0,0 +1,130 @@
+"""
+@CreateTime    : 2024/5/6 14:30
+@Author  : 系统订单服务
+@file for: 订单业务逻辑层
+"""
+import traceback
+import random
+import time
+from typing import Dict, List, Optional, Union
+
+from logger_control import SharingProgramInfo, log_info
```

Pylint (3.3.7)

[error] Unable to import 'logger_control'

(E0401)